tests/e2e/system_test/test_cluster_full_and_recovery.py

```diff
@@ -0,0 +1,348 @@
+        6.Verify Alerts are seen ["CephClusterCriticallyFull", "CephOSDNearFull"]
+        7.Respin 'osd' ,'mgr', 'mon' pods
+        8.Create PVC2 [FS + RBD]
+        9.Verify PVC2 [FS + RBD] are in Pending state
+        10.Create snapshot from PVC1 [FS+RBD]
+        11.Verify snapshots on false state
+        12.Change Ceph full_ratiofrom from 85% to 95%
+        13.Delete  benchmark-operator PVCs
+        14.Change Ceph backfillfull_ratio from 80% to 95%
+        15.Verify PVC2 [FS + RBD]  are moved to Bound state
+        16.Verify snapshots moved from false state to true state
+        17.Restore new pvc from snapshot pvc [RBD + FS]
+        18.Verify checksum PVC1 equal to PVC1_RESTORE
+        19.Change Ceph full_ratiofrom from 95% to 85%
+        20.Change Ceph backfillfull_ratio from 95% to 80%
+
+        """
+        self.count = 0
+        self.banchmark_operator_teardown = False
+        project_name = "test788"
+        self.project_obj = helpers.create_project(project_name=project_name)
+        teardown_project_factory(self.project_obj)
+
+        log.info("Create PVC1 CEPH-RBD, Run FIO and get checksum")
+        pvc_obj_blk1 = pvc_factory(
+            interface=constants.CEPHBLOCKPOOL,
+            project=self.project_obj,
+            size=2,
+            status=constants.STATUS_BOUND,
+        )
+        pod_blk1_obj = pod_factory(
+            interface=constants.CEPHFILESYSTEM,
+            pvc=pvc_obj_blk1,
+            status=constants.STATUS_RUNNING,
+        )
+        pod_blk1_obj.run_io(
+            storage_type="fs",
+            size="1G",
+            io_direction="write",
+            runtime=60,
+        )
+        pod_blk1_obj.get_fio_results()
+        log.info(f"IO finished on pod {pod_blk1_obj.name}")
+        pod_blk1_obj.md5 = cal_md5sum(
+            pod_obj=pod_blk1_obj,
+            file_name="fio-rand-write",
+            block=False,
+        )
+
+        log.info("Create PVC1 CEPH-FS, Run FIO and get checksum")
+        pvc_obj_fs1 = pvc_factory(
+            interface=constants.CEPHFILESYSTEM,
+            project=self.project_obj,
+            size=2,
+            status=constants.STATUS_BOUND,
+        )
+        pod_fs1_obj = pod_factory(
+            interface=constants.CEPHFILESYSTEM,
+            pvc=pvc_obj_fs1,
+            status=constants.STATUS_RUNNING,
+        )
+        pod_fs1_obj.run_io(
+            storage_type="fs",
+            size="1G",
+            io_direction="write",
+            runtime=60,
+        )
+        pod_fs1_obj.get_fio_results()
+        log.info(f"IO finished on pod {pod_fs1_obj.name}")
+        pod_fs1_obj.md5 = cal_md5sum(
+            pod_obj=pod_fs1_obj,
+            file_name="fio-rand-write",
+            block=False,
+        )
+
+        log.info(
+            "Fill the cluster to “Full ratio” (usually 85%) with benchmark-operator"
+        )
+        size = get_file_size(92)
```

Comment:
You created benchmark_fio_factory_fixture but not using it here, any reason?

Reply:
Because I need to delete benchmark-operator at the middle of the test.